api/migrations/20210330134530_authAffiliationOktaUsername.js

```diff
@@ -0,0 +1,38 @@
+const { oktaClient } = require('../auth/oktaAuth');
+const logger = require('../logger')('user seeder');
+
+exports.up = async (knex) => {
+  await knex.schema.table('users', table => {
+    table.string('username').comment('username of user from authentication service');
+  });
+
+  await knex.schema.table('auth_affiliations', table => {
+    table.string('username').comment('username of user from authentication service');
+  });
+
+  const toUpdate = await knex('auth_affiliations').whereNull('username')
+
+  for (let i=0; i<toUpdate.length; i+= 1){
+    // eslint-disable-next-line no-await-in-loop
```

Comment:
I was curious why eslint doesn't like await in loops. Seems like an optimization one probably not necessary at this time, but good to know- https://eslint.org/docs/rules/no-await-in-loop